code/game/objects/items/syndicateReverseCard.dm

```diff
@@ -0,0 +1,55 @@
+
+//Syndie Uno reverse card
+/obj/item/syndicateReverseCard
+	name = "Red Reverse"
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "sc_Red Reverse_uno"
+	desc = "a card."
+	w_class = WEIGHT_CLASS_TINY
+	var/used = FALSE //has this been used before? If not, give no hints about it's nature
+
+/obj/item/syndicateReverseCard/Initialize()
+	..()
+	var/cardColor = pick ("Red", "Green", "Yellow", "Blue") //this randomizes which color reverse you get!
+	name = "[cardColor] Reverse"
+	icon_state = "sc_[cardColor] Reverse_uno"
+
+/obj/item/syndicateReverseCard/update_icon()
+	..()
+	if (used)
+		cut_overlays()
+		add_overlay(image('icons/obj/items_and_weapons.dmi',icon_state = "Reverse_Overlay"))
+
+/obj/item/syndicateReverseCard/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(!(attack_type == PROJECTILE_ATTACK))
+		return FALSE //this means the attack goes through
+	if(istype(hitby, /obj/item/projectile))
+		var/obj/item/projectile/P = hitby
+		if(P?.firer && P.fired_from && (P.firer != P.fired_from)) //if the projectile comes from YOU, like your spit or some shit, you can't steal that bro. Also protects mechs
+			if(ismachinery(P.firer) || isanimal(P.firer) || issilicon(P.firer)) //You can't switcharoo with turrets or simplemobs, or borgs
```

Comment:
Could you not just do ishuman?

Reply:
This is more secure. Covers all the bases

Reply:
I would feel like just limiting it to humans would be more secure imo. Ill talk with alexkar.

Reply:
yeah ur right